docs/api/datetime.md

```diff
@@ -211,15 +215,15 @@ This example shows a datetime with the `date` configuration.
 
 ## Titles
```

Comment:
Both datetimes have the bottom cut off on `md` mode: https://i.gyazo.com/9a18a81166091c4bffb68ed2f6bbdc86.png